web/packages/teleport/src/UnifiedResources/Resources.tsx

```diff
@@ -325,13 +325,16 @@ export function Resources() {
         {attempt.status === 'failed' && resources.length > 0 && (
           <ButtonSecondary onClick={onRetryClicked}>Load more</ButtonSecondary>
         )}
-        {noResults && isSearchEmpty && (
+        {noResults && isSearchEmpty && !params.pinnedResourcesOnly && (
           <Empty
             clusterId={clusterId}
             canCreate={canCreate && !isLeafCluster}
             emptyStateInfo={emptyStateInfo}
           />
         )}
+        {noResults && params.pinnedResourcesOnly && isSearchEmpty && (
+          <NoPinned />
+        )}
```

Comment:
should we go a step further for this:
<img width="1046" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/6e86133c-b988-4c31-a0f1-5d1a5f6883a9">

`no pinned resources found for ana`

Reply:
Yeah that sounds good. will do

Reply:
Updated the copy. However, the 'empty pinned' page without a search has to be changed. Kenny is going to give me some SVG to show/instruct someone how to pin. That'll come tomorrow
https://github.com/gravitational/teleport/assets/5201977/dbed06e5-ba5d-463a-8ee9-614924a43d9e